test/Microsoft.ML.TimeSeries.Tests/TimeSeriesDirectApi.cs

```diff
@@ -84,12 +84,30 @@ public TimeSeriesData(float value)
             }
         }
 
+        private class TimeSeriesDataWithTimestamp
+        {
+            public DateTime Timestamp { get; set; }
+            public Double Value { get; set; }
+        }
+
         private sealed class SrCnnAnomalyDetection
         {
-            [VectorType(3)]
+            [VectorType]
             public double[] Prediction { get; set; }
         }
 
+        //private sealed class SrCnnAnomalyDetectionWithExp
+        //{
+        //    [VectorType(4)]
+        //    public double[] Prediction { get; set; }
+        //}
+
+        //private sealed class SrCnnAnomalyDetectionWithMargin
+        //{
+        //    [VectorType(7)]
+        //    public double[] Prediction { get; set; }
+        //}
+
         [Fact]
```

Comment:
Please delete this

Reply:
deleted

---
In reply to: [418241868](https://github.com/dotnet/machinelearning/pull/4998#discussion_r418241868) [](ancestors = 418241868)

Reply:
> Please delete this

Deleted.